Gemfile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Shouldn't this be covered by: https://github.com/Netflix/spectator-rb/blob/master/.rubocop.yml#L14?